hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALPrettyPrinter.java

```diff
@@ -326,15 +340,19 @@ public void processFile(final Configuration conf, final Path p)
           out.print(GSON.toJson(txn));
         } else {
           // Pretty output, complete with indentation by atomic action
-          out.println(String.format(outputTmpl,
+          if (!outputOnlyRowKey) {
```

Comment:
Why we are removing this info from the output? Is it an additional change not mentioned on the jira original description?

Reply:
yes, I should mention it in jira and I will add it, this option `outputOnlyRowKey` was to print only rowkey but we were still printing additional stuff.